group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -1904,16 +2000,7 @@ public void testReconciliationProcess() {
             new ConsumerGroupHeartbeatResponseData()
                 .setMemberId(memberId1)
                 .setMemberEpoch(11)
-                .setHeartbeatIntervalMs(5000)
```

Comment:
so we are no longer sending a reassignment because we were incorrectly doing so before? (ie they weren't full requests and didn't need new assignments)

Reply:
Yep. This is correct.